src/Makefile.test.include

```diff
@@ -29,8 +29,6 @@ JSON_TEST_FILES = \
   test/data/tx_valid.json \
   test/data/sighash.json
 
-RAW_TEST_FILES = test/data/alertTests.raw
```

Comment:
I think it may be better to empty it instead, in case raw test files are added in the future